sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderAsyncClient.java

```diff
@@ -765,21 +790,28 @@
      * @return the response.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Flux<ByteBuffer> getMapStaticImage(
-            RasterTileFormat format,
-            StaticMapLayer layer,
-            MapImageStyle style,
-            Integer zoom,
-            List<Double> center,
-            List<Double> boundingBox,
-            Integer height,
-            Integer width,
-            String language,
-            LocalizedMapView localizedMapView,
-            List<String> pins,
-            List<String> path) {
-        return this.serviceClient.getMapStaticImageAsync(
-                format, layer, style, zoom, center, boundingBox, height, width, language, localizedMapView, pins, path);
+    public Mono<StreamResponse> getMapStaticImageWithResponse(MapStaticImageOptions options) {
+        return this.getMapStaticImageWithResponse(options, null);
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    Mono<StreamResponse> getMapStaticImageWithResponse(MapStaticImageOptions options, Context context) {
+        BoundingBox boundingBox = options.getBoundingBox();
```

Comment:
is boundingBox inside options mandatory? If not ,this is going to crash when bounding box is null